x/cronos/keeper/msg_server.go

```diff
@@ -15,3 +22,139 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 }
 
 var _ types.MsgServer = msgServer{}
+
+func (k msgServer) ConvertTokens(goCtx context.Context, msg *types.MsgConvertTokens) (*types.MsgConvertResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	acc, err := sdk.AccAddressFromBech32(msg.Address)
+	if err != nil {
+		return nil, err
+	}
+
+	params := k.GetParams(ctx)
+	evmParams := k.GetEvmParams(ctx)
+
+	for _, c := range msg.Amount {
+		switch c.Denom {
+		case params.IbcCroDenom:
+			if params.IbcCroDenom == "" {
+				return nil, sdkerrors.Wrap(types.ErrIbcCroDenomEmpty, "ibc is disabled")
+			}
+
+			// Send ibc token escrow address
+			err = k.bankKeeper.SendCoinsFromAccountToModule(ctx, acc, types.ModuleName, sdk.NewCoins(c))
+			if err != nil {
+				return nil, err
+			}
+			// Compute new amount, because basecro is a 8 decimals token, we need to multiply by 10^10 to make it
+			// a 18 decimals token
+			ten := big.NewInt(10)
+			exponent := ten.Exp(ten, ten, nil)
```

Comment:
How about make it a static variable?

Reply:
thanks, I culdnt make as a constant but var works, fixed